src/index.js

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env node
+
+// Register means every single file will get processed by babel that we import!
+import 'babel/register'
+
+import yaml from 'js-yaml';
+import fsPath from 'path';
+import fs from 'mz/fs';
+import createCommands from './commands';
+
+const CONFIG = 'exhibition.yml';
```

Comment:
> node/npm global install actually works cross platform.. 

heh then you have to manage varying versions of npm/node I think you can guess what this is doing right :) ?